R/parameters_table.R

```diff
@@ -59,14 +59,18 @@ parameters_table <- function(x, pretty_names = TRUE, stars = FALSE, digits = 2,
     x$p <- format(x$p, justify = "left")
   }
 
-  # CI
+  # Main CI
   ci_low <- names(x)[grep("CI_low*", names(x))]
```

Comment:
I think this pattern should be 

```r
grep("^CI_low", names(x))
```

and the others respectively, else `"Eta2_partial_CI_low"` is already caught here.